main/influxdb_watcher.go

```diff
@@ -184,8 +207,14 @@ func main() {
 			break
 		}
 	}
+	for {
+		databaseExists := influxdbBroker.checkDatabaseExists(time.Duration(15) * time.Second)
+		if databaseExists {
+			break
+		}
+		time.Sleep(time.Duration(10) * time.Second)
```

Comment:
just curious why here is 10 sec not 15? haha

Reply:
If anything I should change 15 to 10 to stay consistent across all watcher:
https://github.com/monasca/monasca-watchers/blob/master/main/influxdb_watcher.go#L182
https://github.com/monasca/monasca-watchers/blob/master/main/kafka_watcher.go#L133
but still, it's just going to check and recheck. If I changed these to be configurable, I would want to change it everywhere. Maybe it would be more appropriate to do that in a different patch